userspace/libsinsp/test/CMakeLists.txt

```
@@ -98,22 +99,32 @@ set(LIBSINSP_UNIT_TESTS_SOURCES
 	filter_op_bcontains.ut.cpp
 	filter_op_pmatch.ut.cpp
 	filter_compiler.ut.cpp
-	user.ut.cpp
 	container_info.ut.cpp
 	sinsp_utils.ut.cpp
 	state.ut.cpp
 	eventformatter.ut.cpp
 	savefile.ut.cpp
-	thread_table.ut.cpp
 	public_sinsp_API/event_related.cpp
-	public_sinsp_API/sinsp_logger.cpp
 	"${TEST_PLUGINS}"
-	"${SINSP_PARSERS_SUITE}"
-	"${SINSP_CLASSES_SUITE}"
-	"${SINSP_FILTERCHECKS_SUITE}"
 	"${SCAP_FILES_SUITE}"
 	"${TEST_HELPERS}"
-)
+	)
+
+if(WIN32)
+	list(REMOVE_ITEM LIBSINSP_UNIT_TESTS_SOURCES
```

```suggestion
	list(APPEND LIBSINSP_UNIT_TESTS_SOURCES
```

Aren't these already removed above? 🤔

---

In the process of refactoring I messed up copy and pasting 😓